package.json

```diff
@@ -46,11 +47,12 @@
     "lerna": "^3.20.2",
     "lint-staged": "^10.1.0",
     "prettier": "^2.0.5",
-    "recursive-readdir": "^2.2.2"
+    "recursive-readdir": "^2.2.2",
+    "shx": "^0.3.2"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged && node ./scripts/check-docs-quality"
```

Comment:
It would be better to do this instead:

```json
"lint-staged": {
    "*.{json,md}": [
      "prettier --write"
    ],
 // add here ...
 "*..md": [
      " node ./scripts/check-docs-quality"
    ]
  },
}
```

NOTE: I have not tested this, but the approach would be more inline with how lint-staged works.

Reply:
In that case, simple ```vale``` command is sufficient. Thanks for pointing out.